src/components/toolTip/ToolTip.module.css

```diff
@@ -0,0 +1,49 @@
+:root {
+  --conduction-size-xs: var(--skeleton-size-xs);
+  --conduction-color-white: var(--skeleton-color-white);
+  --conduction-color-black: var(--skeleton-color-black);
+  --conduction-font-size-sm: var(--skeleton-font-size-sm);
+  --conduction-content-font-family: var(--skeleton-content-font-family);
+  --conduction-line-height-xs: var(--skeleton-line-height-xs);
+  --conduction-font-size-4xs: var(--skeleton-font-size-4xs);
+  --conduction-size-xl: var(--skeleton-size-xl);
+}
+
+.wrapper {
+  display: inline-block;
+  position: relative;
+}
+
+.content {
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: var(--conduction-size-xs);
+  color: var(--conduction-color-white);
+  background: var(--conduction-color-black);
```

Comment:
I'd opt to make this more specific: `background-color` instead of just `background`.